easy_infra/utils.py

```diff
@@ -116,7 +116,10 @@ def opinionated_docker_run(
                 "Received an unexpected exit when invoking CLIENT.containers.run() with the following arguments: "
                 + f"{auto_remove=}, {command=}, {detach=}, {environment=}, {image=}, {network_mode=}, {tty=}, {volumes=}, {working_dir=}"
             )
-            sys.exit(response["StatusCode"])
```

Comment:
This is a bugfix that I found when writing tests for this feature.  If it expected a failure but it exited 0, the tests would stop early but exit 0 so they would not show a failure.  See https://github.com/SeisoLLC/easy_infra/actions/runs/3753134806/jobs/6376095210